doc/source/ray-core/actors/task-orders.rst

```diff
@@ -0,0 +1,13 @@
+Actor Tasks Execution Orders
+============================
+
+Synchronous, single-threaded actors execute tasks according
+to their initial submission order. Any submitted task will not
```

Comment:
Add some code examples? I think it's also worth noting that tasks can get reordered between different submitters. For example, if they are waiting on object argument resolution.